forch/endpoint_handler.py

```diff
@@ -13,54 +13,100 @@
 import logging
 import subprocess
 import time
+import threading
+import yaml
 import grpc
 
 from forch.utils import get_logger
-
-try:
-    import daq.proto.session_server_pb2_grpc as server_grpc
-    from daq.proto.session_server_pb2 import SessionParams, SessionProgress
-    SessionServerServicer = server_grpc.SessionServerServicer
-    PROTO_LOADED = True
-except ImportError:
-    PROTO_LOADED = False
-
-    class SessionServerServicer:
-        """Dummy class for weak import"""
+import forch.proto.endpoint_server_pb2_grpc as server_grpc
+from forch.proto.endpoint_server_pb2 import Endpoint
+from forch.proto.shared_constants_pb2 import Empty
 
 DEFAULT_SERVER_PORT = 50111
 DEFAULT_BIND_ADDRESS = '0.0.0.0'
 DEFAULT_VXLAN_PORT = 4789
-DEFAULT_VXLAN_VNI = 0
-
+BASE_T1SW_PORT = 29
 VXLAN_CMD_FMT = 'ip link add %s type vxlan id %s remote %s dstport %s srcport %s %s nolearning'
-
+T1_SW1 = 'nz-kiwi-t1sw1'
```

Comment:
Add TODO that this should be configurable.  It's only a constant used by testing, but that's all this code is currently used for -- so I don't think it's worth actually making it configurable now, but at least acknowledge with TODO.

Or, at least investigate if it's reasonably trivial to extract this from the structural_config_file below.  I know it's a bit hacky, but I think a heuristic like "identify switch with TAP port description -- choose the one with lowest dp_id" might work well.

Up to you -- don't think it'll actually matter in the long run so do whatever.